### PR TITLE
Added Rule "Do you use Nx to manage your huge JavaScript solution"

### PR DESCRIPTION
**Pull request by christoment**
feat:
- Added rule "Tools - Do you use Nx to manage your huge JavaScript solution?"
- Link the rule to Angular and React group

### CODE REVIEW

File: rules/manage-javascript-projects-with-nx/rule.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+One of the main problems working on a huge monorepo solution is usually the development experience and the build time.
+[Nx](https://nx.dev/) is one of many tools that can improve this experience in JavaScript projects.
+
+<!--endintro-->
+
+The amount of code that needs to be processed by the compiler scales proportionally with the solution size. Hence, the compile time will grow naturally as the solution grows in size.
+This surely affects both the development experience and the team's velocity, leaving both developers and stakeholders unhappy.
+
+[Nx](https://nx.dev/) is a JavaScript build system that aims to make developing on monorepo solution easier and faster.
```
Comment (tiagov8) [inline rules/manage-javascript-projects-with-nx/rule.md:23]: @christoment no need to repeat the link
https://ssw.com.au/rules/do-you-avoid-redundant-linking-aka-single-link-to-single-location/

File: rules/manage-javascript-projects-with-nx/rule.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+Disadvantages:
+:::
+
+* Additional external dependency to be maintained
+* Learning curve
+* Only supports JavaScript projects
+
+
+Consider using Nx in a project when your solution:
+* is a JavaScript monorepo
```
Comment (tiagov8) [inline rules/manage-javascript-projects-with-nx/rule.md:53]: @christoment 

Please start this list items with Capitalized words

Comment (christoment) [reply rules/manage-javascript-projects-with-nx/rule.md]: Done :)

File: rules/manage-javascript-projects-with-nx/rule.md (hunk 1 above)
Comment (christoment) [reply rules/manage-javascript-projects-with-nx/rule.md]: Done :)

**Review by tiagov8 — APPROVED** [review]

**Pull request merged**